src/main/webapp/js/userMapData.json

```diff
@@ -1,6 +1,9 @@
 {
-  "lastUpdated": "31 January 2017",
+  "lastUpdated": "15 March 2017",
   "institutes": {
+  	 "Abu Dhabi": [ 	
+  	   "Khalifa University"
+  	  ],
```

Comment:
Looks like there is an indentation problem here. Also, see if the same university is listed under UAE.